src/note-utils.ts

```diff
@@ -18,6 +19,8 @@ export const findNoteInPRBody = (body: string): string | null => {
     notes = notes.replace(/<!--.*?-->/g, '');
   }
 
+  debug(`Found Notes: ${notes.trim()}`);
```

Comment:
Can we wrap `notes.trim()` in `JSON.stringify` so that special characters like newlines are easily visible in the logs